frontend/src/reducer/currentWorkspace.js

```diff
@@ -61,6 +62,21 @@ export default function currentWorkspace (state = defaultWorkspace, action) {
         }))
       }
 
+    case `${ADD}/${WORKSPACE_MEMBER}`:
+      return {
+        ...state,
+        memberList: [
+          ...state.memberList,
+          {
+            id: action.member.user_id,
+            publicName: action.member.user.public_name,
```

Comment:
`publicName` is gone from this endpoint, you should remove it.

Reply:
I'd say this is up to the PR for the username